pkgs/shells/yash/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, lib, fetchurl, gettext, ncurses }:
+
+stdenv.mkDerivation rec {
+  pname = "yash";
+  version = "2.52";
+
+  src = fetchurl {
+    url = "https://osdn.net/dl/yash/yash-${version}.tar.xz";
+    sha256 = "sha256:1jdmj4cyzwxxyyqf20y1zi578h7md860ryffp02qi143zpppn4sm";
+  };
+
+  meta = with lib; {
+    description =
+      "Yet another shell, is a POSIX-compliant command line shell written in C99 (ISO/IEC 9899:1999)";
```

A more concise description:

```suggestion
    description = "Yet another POSIX-compliant shell";
```